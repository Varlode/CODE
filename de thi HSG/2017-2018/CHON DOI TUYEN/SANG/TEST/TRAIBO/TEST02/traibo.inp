2
2 2
0
1000000
5 3
0
200000
400000
600000
800000