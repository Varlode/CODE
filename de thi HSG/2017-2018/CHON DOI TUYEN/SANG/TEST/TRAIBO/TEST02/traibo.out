1000000
400000